.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/psf/black
-    rev: 22.10.0
+    rev: 22.8.0
```

Comment:
Why did you downgraded black?

Reply:
I have tested it manually it resolve the issue of installing black on Centos 7 and Ubuntu 18.04